Plugins/ServerPlugin.cpp

```diff
@@ -242,31 +242,81 @@ void CompilerClient::UpdateLoop() {
 ServerPlugin::ServerPlugin(MainWindow& mainWindow) : RGMPlugin(mainWindow) {
   // create a new child process for us to launch an emake server
   process = new QProcess(this);
+  
+  connect(process, &QProcess::errorOccurred, [=](QProcess::ProcessError error) { 
+    qDebug() << "QProcess error: " << error << endl; 
+  });
+  
+  //TODO: Make all this stuff configurable in IDE
+  QProcessEnvironment env = QProcessEnvironment::systemEnvironment();
+  
+  #ifdef _WIN32
+  // FIXME: this is just an approximate guess on how to get emake running outside a msys shell and currently causes emake to crash

```

Comment:
@RobertBColton I need your help debugging this. I can just leave it broken for now and have josh merge if you don't want to look now though